### DISCUSSION TITLE
Cannot add additional php packages in Docker image

### DISCUSSION THREAD

**Discussion by hatzisn** [TimWolla/docker-adminer#156] [Q&A]
Hi I tried with this Dockerfile and I get the error that follows it immediately. Which are the root's username/password?

Dockerfile:

FROM adminer:latest
RUN apt-get install pdo_dblib
RUN apt-get install oci8
RUN apt-get install interbase
RUN apt-get install mongodb

Error:
E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?

**Comment by TimWolla** [comment]
Use `USER root` before your `apt` invocation and `USER adminer` afterwards.
